oaknational/Oak-Web-Application
# fix: transcripts for canonical lessons [LESQ-835]

## Description

Music year: 2019

[Ticket](https://www.notion.so/oaknationalacademy/Canonical-lessons-do-not-display-transcripts-e65b1527ada8406c939eccc396d6c1de?pvs=4)
- move logic to fetch and transform video transcripts into lesson overview query used by canonical lessons as well

## Issue(s)

Fixes #
canonical lessons not having video transcripts

## How to test

1. Go to https://deploy-preview-2545--oak-web-application.netlify.thenational.academy/teachers/lessons/developing-a-model-for-atoms#video
3. You should see trasncript for the video

## Screenshots

How it used to look (delete if n/a):
<img width="600" alt="Screenshot 2024-06-27 at 14 03 17" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/80bc787e-e238-4c63-a6c8-0a12b3e5778f">

How it should now look:
<img width="600" alt="Screenshot 2024-06-27 at 14 03 50" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/df92284e-d361-4769-a51b-62734b4317a1">


## Checklist

- [x] New lessons have video transcripts on canonical pages
- [x] Legacy lessons still have transcripts on canonical lesson pages
- [x] All non-canonical lesson pages with video should still have a transcript if they had one before
